cardstack/cardstack
# Show recent cards only for a given device in deployed apps

Closes #1487 

The goal is that when we send people a link to demo this, they should not see other people's cards. Note that this has nothing to do with authorization. All cards in a demo are public-readable by anyone.

## Added:
- `csCreatedBy` string to the Card API
- `DEVICE_CARDS_ONLY` env var that is passed into the app.
- When ^^^ is true, the Library will fetch the recent cards associated with the given device, not all recent cards
- `cardLocalStorage` service, which sets a semi-random string for `Device` in the browser local storage
- The application controller sets up the cardLocalStorage
- Tests

## How to test
- in config/environment.js, set deviceCardsOnly to true
- run the app. Make a card. Open the library and see only your card there. Refresh and see it's still there. Clear local storage. See that the card doesn't show up in the library. Make a new one, and see it show up.
